README.md

```diff
@@ -21,6 +21,21 @@ At the moment these packages are supported:
 composer require rector/rector --dev
 ```
 
+# Install on PHP < 7.1
+
+You must have separated environment with PHP 7.1 (for example in Docker container). When you have it then run next command. The reason for this difficulty setup is your dependencies are only PHP <7.1 compatible and you cannot add 7.1.x package.
```

Comment:
- run next => run following
- "The reason for this difficulty setup is your dependencies are only PHP <7.1 compatible and you cannot add 7.1.x package." 
   - I think this information is already included in the title.

Reply:
Fixed